libs/math/include/math/tensor.hpp

```diff
@@ -2599,13 +2599,12 @@ bool Tensor<T, C>::AllClose(Tensor const &o, Type const &relative_tolerance,
     ++it1;
     ++it2;
 
-    T abs_e1 = e1;
-    fetch::math::Abs(abs_e1, abs_e1);
-    T abs_e2 = e2;
-    fetch::math::Abs(abs_e2, abs_e2);
-    T abs_diff = e1 - e2;
-    fetch::math::Abs(abs_diff, abs_diff);
-    T tolerance = std::max(absolute_tolerance, std::max(abs_e1, abs_e2) * relative_tolerance);
+    T abs_e1, abs_e2, abs_diff;
```

Comment:
i believe this might be contrary to our style guide (I think we are supposed to have one declaration per line for clarity)

Reply:
fixed.